DQM/SiStripCommissioningSources/plugins/tracking/SiStripFineDelayHit.cc

```diff
@@ -419,10 +419,10 @@ SiStripFineDelayHit::produce(edm::Event& iEvent, const edm::EventSetup& iSetup)
          else if(((layerCode>>6)&0x3)==2) subdet = StripSubdetector::TID;
          else if(((layerCode>>6)&0x3)==3) subdet = StripSubdetector::TEC;
          int32_t layerIdx = (layerCode&0xF)*(((layerCode>>4)&0x3) ? -1 : 1);
```

Comment:
The block of code above this looks like something we need to have a look into as well... I'm a bit confused by `>>16` and then `>>6`, while `DetId` says the subdet ID is `(id_>>25)&0x7`. Does anyone have an idea?